recipe-client-addon/lib/Addons.jsm

```diff
@@ -47,9 +48,9 @@ this.Addons = {
    * @async
    * @returns {Array<SafeAddon>}
    */
-  getAll: async function get(addonId) {
+  async getAll(addonId) {
     const addons = await AddonManager.getAllAddons();
-    return addons.map(Addons.serializeForSandbox);
+    return addons.map(Addons.serializeForSandbox.bind(this));
```

Comment:
This looks odd - `Addons` and `this` are equivalent, it looks like, but it would be more obvious if we bound to the same thing we accessed the method on (so either `Addons.serializeForSandbox.bind(Addons)` or `this.serializeForSandbox.bind(this)`).